README.md

```diff
@@ -1,59 +1,162 @@
 
 ```sh
-mvn spring-boot:run -e -X
+mvn spring-boot:run
 ```
 
-You can launch your browser and go to the [Swagger start page](https://localhost:8080/swagger-ui/index.html).
+### Using Docker
 
-## Running The Tests
+Build the Docker image
 
-To run the standard integration tests (which requires a running Couchbase Server), use the following command:
+```sh
+docker build -t java-springboot-quickstart .
+```
+
+Run the Docker image
 
 ```sh
-mvn verify
+docker run -d --name springboot-container -p 8080:8080 java-springboot-quickstart
 ```
 
-## Project Setup Notes
+Note: The `application.properties` file has the connection information to connect to your Capella cluster. These will be part of the environment variables in the Docker container.
+
+### Verifying the Application
+
+Once the application starts, you can see the details of the application on the logs.
+
+![Application Startup](app_startup.png)
+
+The application will run on port 8080 of your local machine (http://localhost:8080). You will find the interactive Swagger documentation of the API if you go to the URL in your browser. Swagger documentation is used in this demo to showcase the different API endpoints and how they can be invoked. More details on the Swagger documentation can be found in the [appendix](#swagger-documentation).
+
+![Swagger Documentation](swagger_documentation.png)
+
+## Running Tests
+
+To run the integration tests, use the following commands:
+
+```sh
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.AirlineIntegrationTest
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.AirportIntegrationTest
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.RouteIntegrationTest
+
+You can also run all the tests using the following command:
+mvn test
+```
+
+## Appendix
+
+### Data Model
+
+For this quickstart, we use three collections, `airport`, `airline` and `routes` that contain sample airports, airlines and airline routes respectively. The routes collection connects the airports and airlines as seen in the figure below. We use these connections in the quickstart to generate airports that are directly connected and airlines connecting to a destination airport. Note that these are just examples to highlight how you can use SQL++ queries to join the collections.
+
+![travel-sample data model](travel_sample_data_model.png)
+
+### Extending API by Adding New Entity
+
+If you would like to add another entity to the APIs, these are the steps to follow:
+
+- Create the new entity (collection) in the Couchbase bucket. You can create the collection using the [SDK](https://docs.couchbase.com/sdk-api/couchbase-java-client-3.5.2/com/couchbase/client/java/Collection.html#createScope-java.lang.String-) or via the [Couchbase Server interface](https://docs.couchbase.com/cloud/n1ql/n1ql-language-reference/createcollection.html).
```

Comment:
The SDK reference is wrong. It should be the management API, not the collection API

Reply:
Given the Couchbase Capella Management API (v4.0)